services/capture/src/main/resources/log4j2.yml

```diff
@@ -0,0 +1,60 @@
+Configuration:
+  properties:
+    property:
+      - name: filename
+        value: logs/app.log
+      - name: bootstrapServers
+        value: ${env:KAFKA_BOOTSTRAP_SERVERS}
+      - name: topic
+        value: ${env:KAFKA_LOGS_TOPIC}
+
+  status: warn
+
+  appenders:
+    Console:
+      name: LogToConsole
+      PatternLayout:
+        Pattern: "[%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%-7t] %c{1}:%L - %m%n"
+
+    #File:
+    #  name: File
+    #  fileName: logs/app.log
+    #  PatternLayout:
+    #    Pattern: "%d %p %C{1.} [%t] %m%n"
+
+    RollingFile:
```

Comment:
Look at syndication service.  I think I needed to comment this stuff out because it won't work in Openshift.

Reply:
I commented it out to match the syndication config.